### PR TITLE
use different example for luhn's

### PR DESCRIPTION
**Pull request by curiouskiwi**
to avoid confusion about "start from 2nd to last digit", changed example to use a 16-digit card.

### CODE REVIEW

**Comment by dmalan** [comment]
@kzidane mind taking a look too?

File: credit/README.md [hunk 1]
```diff
@@ -12,29 +12,29 @@ So what's the secret formula?  Well, most cards use an algorithm invented by Han
...
 
 1. For the sake of discussion, let's first underline every other digit, starting with the number's second-to-last digit:
    
-   3<u>7</u>8<u>2</u>8<u>2</u>2<u>4</u>6<u>3</u>1<u>0</u>0<u>0</u>5
+   <u>4</u>0<u>0</u>3<u>6</u>0<u>0</u>0<u>0</u>0<u>0</u>0<u>0</u>0<u>1</u>4
 
    Okay, let's multiply each of the underlined digits by 2:
 
-   7•2 + 2•2 + 2•2 + 4•2 + 3•2 + 0•2 + 0•2
+   4•2 + 0•2 + 6•2 + 0•2 + 0•2 + 0•2 + 0•2 + 1•2
```
Comment (kzidane) [inline credit/README.md]: @dmalan @curiouskiwi it doesn't really matter but just for consistency, should we start here with the second to last digit?

**Review by kzidane — APPROVED** [review]
Other than https://github.com/cs50/labs/pull/1/files#r217874708, looks good to me.

**Comment by kzidane** [comment]
PS, should we change the credit card number in the examples below too?

File: credit/README.md (hunk 1 above)
Comment (curiouskiwi) [reply credit/README.md]: The original version had them left to right, so I left it that way.  But showing from last to first might be even clearer. I'll update.

**Pull request merged**